port_ocean/log/handlers.py

```diff
@@ -8,18 +9,32 @@
 from logging.handlers import MemoryHandler
 from traceback import format_exception
 from typing import Any
-
 from loguru import logger
 
 from port_ocean import Ocean
 from port_ocean.context.ocean import ocean
 
 
+def _remove_posix_paths(extra: dict[str, Any]) -> dict[str, Any]:
```

Comment:
to prevent a case where something might be cyclic, please add a hard limit (and log an error or something if it reached)